stairway/src/main/java/bio/terra/stairway/impl/ProgressMetersImpl.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.stairway.impl;
+
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.ProgressMeterData;
+import bio.terra.stairway.ProgressMeterReader;
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class ProgressMetersImpl implements ProgressMeterReader {
+  /** FlightMap key for persistedStateMap for storing the progress meters */
+  public static final String STAIRWAY_PROGRESS_METERS = "StairwayProgressMeters";
```

Comment:
Does stairway reserve keys that begin with a specific prefix? If so, it should check for this prefix to prevent a caller from overriding a property that stairway "owns".

Reply:
Good idea. I'll implement that.